assets/promptflow/evaluators/models/protected-material-evaluator/spec.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Protected-Material-Evaluator
+path: ./
+properties:
+  is-promptflow: true
+  is-evaluator: true
+  show-artifact: true
+  _default-display-file: ./evaluator/flow/prompt.jinja2
```

Comment:
same question: is there jinja2 file for safety metrics?

Reply:
I will remove these - I realize they're not needed here. We'll use the description.md